app/models/intake_site_drop_off.rb

```diff
@@ -116,7 +116,7 @@ def formatted_phone_number
 
   # Returns the phone number in the E164 standardized format, e.g.: "+15105551234"
   def standardized_phone_number
-    Phonelib.parse(phone_number).e164
+    Phonelib.parse(phone_number, "US").e164
   end
```

Comment:
This seems like an improvement. Should it be a module or an external class? Should we use this to improve our phone number parsing in our drop off forms or anywhere else?

Reply:
Agreed. I think we could perhaps make a phone number wrapper class that includes these responsibilities as well as the /questions/phone-number edit page's "Add the country code if necessary" logic. I'll make a gardening story for it.